root/scripts/recipepage.js

```diff
@@ -142,6 +142,14 @@ async function init() {
   } else {
     button.style = 'color:grey';
   }
+
+  (function(d, s, id) {
+    var js, fjs = d.getElementsByTagName(s)[0];
```

Comment:
Codacy has a fix for the issue: [Expected variable declaration to be on a new line.](https://app.codacy.com/gh/cse110-fa21-group34/rocketrecipes/pullRequest?prid=8487889)
```suggestion
    var js, 
```